fdbbackup/backup.actor.cpp

```diff
@@ -1054,9 +1056,12 @@ static void printBackupUsage(bool devhelp) {
 	       "containing no data at or after a\n"
 	       "                 version approximately NUM_DAYS days worth of versions prior to the latest log version in "
 	       "the backup.\n");
+	printf("  --query-restore-snapshot-version VERSION\n"
+	       "                 For query operations, set snapshot version for restoring a backup. Set -1 for maximum\n"
```

Comment:
" For query operations, set snapshot version for restoring a backup. " ->
" For query operations, set the snapshot version, inclusive, used to restore a backup. "

for better readability

Reply:
" Set -1 for maximumrestorable version (default) and -3 for minimum restorable version." 
Change to
"
Set -1 to use the latest valid snapshot,
Set -3 to use the oldest valid snapshot. 
"